docs/topics/mta-web-edit-application.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-edit-application_{context}"]
+= Opening and editing an application
+
+You open and edit an existing application to the *Application Inventory* for changes in the assessment and analysis.
+
+.Prerequisites
+
+* Log in to a server running {ProductShortName}
+
+.Procedure
+
+. In *Migration* view, click *Application Inventory*.
+. Click the row of an application to open a summary of the application's current settings.
++
+.View and edit application
+image::mta-web-app-details-01.png[View and edit application]
+
+. To change application settings, click {icon-edit.png} to open the application settings, and review the application settings. See previous procedure for a list of application settings.
+. If you changed any application settings, then click *Save*, otherwise click *Cancel*.
+
+// [Verification]
```

Comment:
This file does not appear in the preview as it is not included in the `master.adoc`